app/schemas/event_schema.rb

```diff
@@ -50,4 +50,9 @@ class EventSchema < RubyLLM::Schema
   string :featured_color,
     description: "CSS text color for featured cards",
     required: false
+
+  object :coordinates, description: "Geographic coordinates", required: false do
+    number :latitude, description: "Latitude coordinate"
+    number :longitude, description: "Longitude coordinate"
+  end
```

Comment:
Is there a reason why we copy this over to the `event.yml` file? I feel like keeping it in the `venue.yml` file helps to avoid duplication

Reply:
I was wondering if we should just write this to a `venue.yml` if there is no `venue.yml` yet instead of putting it in the `event.yml`. Just so we only have one location where coordinates live.

```yaml
coordinates:
  latitude: 35.59816209289096
  longitude: -82.55246525892764
```

Or is that too confusing?

Reply:
Yeah, fair enough! Was not trying to be annoying, I just wanted to explore if there's a better way to go about it! 👍🏼 

Let's continue with `event.yml` approach then 👍🏼